backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetWaitingTimeResponse.java

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.proto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GetWaitingTimeResponse {
+    private long waitingTimePerCustomer;
```

Comment:
Priyanka mentioned that there is no need to explicitly mention things like perCustomer while naming the variables..

Reply:
I added to avoid any confusion and to specify that the waitingTime is not yet multiplied with customersInQueue. Maybe our hosts can guide us here. Thanks Bavanya for highlighting this.

Reply:
waitingTimePerCustomer is fine here for the reason Karishma stated above.
Also please add a comment above the member variable: 
"The approximate waiting time in minutes per customer."

Reply:
Okay sure.